bitnami/elasticsearch/README.md

```diff
@@ -70,6 +70,11 @@ The following table lists the configurable parameters of the Elasticsearch chart
 | `config`                                          | Elasticsearch node custom configuration                                                                                                                   | ``                                                           |
 | `extraVolumes`                                    | Extra volumes                                                                                                                                             |                                                              |
 | `extraVolumeMounts`                               | Mount extra volume(s),                                                                                                                                    |                                                              |
+| `initdbScripts`                                   | Dictionary of initdb scripts                                                                                                                              | `nil`                                                        |
+| `initdbScriptsConfigMap`                          | ConfigMap with the initdb scripts (Note: Overrides `initdbScripts`)                                                                                       | `nil`                                                        |
```

Comment:
In other charts we also allow init scripts as secrets in case they want to put sensitive data there. Using subfolders both can co-exist. Example: kong

https://github.com/bitnami/charts/blob/master/bitnami/kong/templates/deployment.yaml#L166

Reply:
OK, added.